docs/Configuration.md

```diff
@@ -369,6 +369,22 @@ You can collect coverage from those files with setting `forceCoverageMatch`.
 }
 ```
 
+### `freezeCoreModules` [boolean]
+
+Default: `false`
+
+Prevents overriding node's core module methods so to prevent memory leaks. 
+
+Attempt to override such a property will fail silently.
```

Comment:
our > 10K test suite runs successfully with these changes, so i guess that in most cases, there is no actual need/use of these overrides.